### PR TITLE
tests: fix wrong test name and harden lease test

### PR DESCRIPTION
**Pull request by BusyJay**
@siddontang @hhkbp2 PTAL

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: tests/raftstore/test_lease_read.rs [hunk 1]
```diff
@@ -92,14 +92,19 @@ fn test_renew_lease<T: Simulator>(cluster: &mut Cluster<T>) {
     cluster.cfg.raft_store.raft_log_gc_threshold = 100;
     // Increase the Raft tick interval to make this test case running reliably.
     cluster.cfg.raft_store.raft_base_tick_interval = 50;
+    cluster.cfg.raft_store.raft_election_timeout_ticks = 90;
```
Comment (siddontang) [inline tests/raftstore/test_lease_read.rs]: why 90 here?

File: tests/raftstore/test_lease_read.rs [hunk 2]
```diff
@@ -92,14 +92,19 @@ fn test_renew_lease<T: Simulator>(cluster: &mut Cluster<T>) {
...
     let election_timeout = Duration::from_millis(cluster.cfg.raft_store.raft_base_tick_interval) *
                            cluster.cfg.raft_store.raft_election_timeout_ticks as u32;
+
     let node_id = 1u64;
     let store_id = 1u64;
     let peer = new_peer(store_id, node_id);
     cluster.run();
 
+    // Wait for election
+    thread::sleep(election_timeout);
```
Comment (hhkbp2) [inline tests/raftstore/test_lease_read.rs]: Why adding this line?

**Comment by hhkbp2** [comment]
Rest LGTM

File: tests/raftstore/test_lease_read.rs (hunk 1 above)
Comment (BusyJay) [inline tests/raftstore/test_lease_read.rs]: To make lease longer.

File: tests/raftstore/test_lease_read.rs (hunk 2 above)
Comment (BusyJay) [inline tests/raftstore/test_lease_read.rs]: Otherwise put operation may be timeout because election has not completed yet.

**Comment by ngaut** [comment]
PTAL

File: tests/raftstore/test_lease_read.rs (hunk 2 above)
Comment (siddontang) [inline tests/raftstore/test_lease_read.rs]: seem we should wait 2 * election_timeout to guarantee election start.

Comment (BusyJay) [inline tests/raftstore/test_lease_read.rs]: Put operation has waiting for it already.

**Comment by BusyJay** [comment]
@hhkbp2 PTAL

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**